docs/reference/class/MongoDBCollection.txt

```diff
@@ -71,6 +72,7 @@ Methods
    /reference/method/MongoDBCollection-drop
    /reference/method/MongoDBCollection-dropIndex
    /reference/method/MongoDBCollection-dropIndexes
+   /reference/method/MongoDBCollection-estimatedDocumentCount
```

Comment:
Building the docs produced the following warning:

> source/reference/class/MongoDBCollection.txt:59: WARNING: toctree contains reference to nonexisting document u'reference/method/MongoDBCollection-estimatedDocumentCount'

Looks like a typo in the filename for `MongoDBCollection-estimateDocumentCount.txt` (missing a "d" before "Document").